REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.6
-BinDeps 0.6.0
+BinaryProvider 0.3.2
```

Comment:
what's necessary from 0.3.2 that wasn't in 0.3.0 (or earlier?)

Reply:
There have been bug fixes added lately and it's the only version which has been tested so it's the only version known to work. Feel free to test prior versions and submit a PR if you can verify that it works for prior versions.

Reply:
are the bug fixes needed here? that's not what minimum versions are for. they are lowest version that should work based on features being used, not version that has been tested.

Reply:
Hello? You shouldn't release things that are just wrong.